src/Resources/views/macros/components.html.twig

```diff
@@ -72,10 +82,18 @@
 
             {% if 'twig' in outputs %}
                 {% set json = component_properties|ui_printable_arguments %}
-                <pre class="c-component__code">{{ '{{ ' }}ui.component('{{ slug }}', {{ json }}{{ ') }}' }}</pre>
+                <pre class="c-component__code">{{ '{% ' }}include '{{ template }}' with {{ json }} only{{ ' %}' }}</pre>

```

Comment:
Is het niet verwarrend dat hier nu template gebruikt wordt ipv component?

Reply:
Het woordje `template` is hoe twig dit noemt.  Deze PR gaat er vooral om dat we meer conform de twig standaard werken ipv ons eigen schilletje daar omheen, dus... zou het lekker template gaan noemen voortaan :stuck_out_tongue:

Reply:
Okee, prima, het is vooral om eenduidigheid te scheppen. Ik lees voor sommige dirs en onderdelen het woord 'template' terwijl ze soms andere dingen betekenen. Voelt altijd wat verwarrend aan.

Reply:
Je doet een include van `'components/foor/bar.html.twig'`, dat is een template. De component aanroepen zullen nu nauwelijks verschillen van gewone includes zoals we die al hadden. De twee belangrijkste verschillen zullen zijn dat component templates in de map `components/` staan en dat je altijd _only_ gebruikt (in `with { ... } only`).

Deze variabele heette eerst "slug", maar dat is het nu niet meer. Het is een template incl. volledige pad en `.html.twig` extensie.